.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "crates/symbolicator-crash/sentry-native"]
+	path = crates/symbolicator-crash/sentry-native
+	url = https://github.com/getsentry/sentry-native
```

Comment:
oh no, do does this force us to use submodules?  i really don't get on with them

Reply:
Fraid so. I guess extracting this into its own crate would also save symbolicator from this fate?